weather

```diff
@@ -0,0 +1,16 @@
+api_key = 'eb3f523c4bedada94d5846340c5dbde7'
+one_call = 'https://api.openweathermap.org/data/2.5/onecall?lat=35.6870&lon=-105.9378&exclude=hour,daily&units=metric'
+headers = {'Content-Type': }
+path = '/Users/anhmai/Desktop/weather\ api '
```

Comment:
We should take care to make our code a bit more portable. Try to use a relative path, or receive the inputs from the user in another way.

Reply:
I'm also not sure if this is referring to a file or not. Doublecheck to see if this works the way you intend it to!